README.md

```diff
@@ -1,6 +1,29 @@
-# Eclipse GLSP Examples [![build-status](https://img.shields.io/jenkins/build?jobUrl=https%3A%2F%2Fci.eclipse.org%2Fglsp%2Fjob%2Feclipse-glsp%2Fjob%2Fglsp-examples%2Fjob%2Fmaster%2F)](https://ci.eclipse.org/glsp/job/eclipse-glsp/job/glsp-examples/job/master) [![build-status-server](https://img.shields.io/jenkins/build?jobUrl=https://ci.eclipse.org/glsp/job/deploy-npm-glsp-examples/&label=npm)](https://ci.eclipse.org/glsp/job/deploy-npm-glsp-examples) [![build-status-server](https://img.shields.io/jenkins/build?jobUrl=https://ci.eclipse.org/glsp/job/deploy-m2-glsp-examples/&label=maven)](https://ci.eclipse.org/glsp/job/deploy-m2-glsp-examples)
+# Eclipse GLSP Examples ![build-status](https://img.shields.io/jenkins/build?jobUrl=https%3A%2F%2Fci.eclipse.org%2Fglsp%2Fjob%2Feclipse-glsp%2Fjob%2Fglsp-examples%2Fjob%2Fmaster%2F)
+
+This repository contains code examples that demonstrates how build diagram editors with the [Graphical Language Server Platform (GLSP)](https://github.com/eclipse-glsp/glsp). The examples are focused on the integration of GLSP editors with the cloud-based [Eclipse Theia IDE](https://github.com/theia-ide/theia) using the [GLSP theia integration](https://github.com/eclipse-glsp/glsp-theia-integration).
```

Comment:
Typos:
- "[...] code examples that demonstrates [...]" -> "[...] code examples that demonstrate [...]" (no 's' on demonstrates)
- "[...] demonstrate how build diagram editors [...]" -> "[...] demonstrate how to build diagram editors [...]" (missing 'to')
- "[...] using the GLSP theia integration." -> "[...] using the GLSP Theia integration." (capitalize 'Theia')